app/assets/stylesheets/media.scss

```diff
@@ -374,7 +374,21 @@
       font-family: "Roboto", "Helvetica", "Arial", sans-serif;
     }
 
+    .logoContainer {
+      padding-top: 16px;
```

Comment:
can we use relative units for padding (rems) as we do elsewhere in this file?

Reply:
sure, will do

Reply:
Done.